### PR TITLE
Added linting to functional tests

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
I have added linting to run consistently on the functional tests.

### How to review 
Is all the linting fixed and passing in CI

### CODE REVIEW

File: gulp/scripts.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ export function lint() {
   // either the tests or app code fail to throw lint errors
```
Comment (shahi645) [inline gulp/scripts.js:100]: @jonnyshaw89 What does this lint function get used for now ?

Comment (jonnyshaw89) [reply gulp/scripts.js:100]: https://github.com/ONSdigital/eq-survey-runner/pull/1407/files#diff-ba210a9157252cc983268fdc3aa3ed52R116

**Review by shahi645 — APPROVED** [review]
Top notch Linting my friend!

**Review by yoganandkunche — APPROVED** [review]

**Comment by yoganandkunche** [comment]
@jonnyshaw89 rebase?

**Pull request merged**